analyzer/analyzer.go

```diff
@@ -10,32 +10,40 @@ import (
 	"github.com/manuelarte/embeddedstructfieldcheck/internal"
 )
 
-const ForbidMutexName = "forbid-mutex"
+const (
+	EmptyLineCheck   = "empty-line"
+	ForbidMutexCheck = "forbid-mutex"
+)
```

Comment:
You are updating the name of an exported constant.

It could be considered as a breaking change.

I understand why you are doing it.

So here you have multiple ways to sort it out:
- assuming the breaking change
- keep the old constant, make it use the new constant, but mark it as // Deprecated
- question yourself about the need for this constants to be exported or not

Side remark, I would have chosen a common prefix instead of common suffix. It helps with IDE completion.

I would have used

        checkEmptyLine   = "empty-line"
        checkForbidMutex = "forbid-mutex"


Also a `type checkOption string` could have been used, but here it might be nitpicking and overkill.

Feel free to ignore, of course. But at least, consider the breaking change